canlink/json.go

```diff
@@ -0,0 +1,40 @@
+package canlink
+
+import (
+	"strconv"
+
+	"encoding/json"
+	"go.uber.org/zap"
+)
+
+// Json object provides utilities for writing frames to trace files in json format
+type Json struct {
```

Comment:
this struct, `NewJson` and `GetFileExtension` seem very repetitive. Does the struct need to store a fileextension field, or can `GetFileExtension` just return `"jsonl"`